mobile/src/components/Navigator.tsx

```diff
@@ -5,6 +5,7 @@ import { AuthContext } from "../contexts/AuthContext";
 import Chess from "../screens/Chess";
 import ForgotPassword from "../screens/ForgotPassword";
 import HomeScreen from "../screens/HomeScreen";
+import LoadingScreen from "../screens/LoadingScreen";
 import LoginScreen from "../screens/LoginScreen";
 
 const Stack = createNativeStackNavigator();
```

Comment:
This is where the navigation type that I mentioned in the other comment should be used.

Reply:
resolved, added a Stack Params type